src/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://nc-news-project-q2e5.onrender.com/api",
+});
+
+const allArticles = instance.get("/articles");
+
+export { allArticles };
```

Comment:
This would be more descriptive if it was called getAllArticles. Firstly, imagine if we had an endpoint to delete all articles (we won't, but that doesn't mean it's impossible). Secondly, all functions should be named after an action, and variables should be named after a noun.